src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -60,7 +91,18 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public List<String> getTopThreeWeatherCondition7(){
-        return null;
+
+        Map countByRoadConditions = getCountByRoadSurfaceCondition7();
```

Comment:
There's a difference between road conditions and weather conditions. Road conditions are "slippery, iced, dry, etc" Weather conditions are "rainy, snowy, windy, etc"